tests/plugins/toolsets/datadog/rds/test_datadog_rds_integration.py

```
@@ -0,0 +1,479 @@
+"""
+Integration tests for Datadog RDS toolset.
+
+These tests use mocked responses to test the toolset functionality
+without requiring actual Datadog API access.
+"""
+
+import json
+from unittest.mock import Mock, patch
```

Ruff (0.12.2)

`unittest.mock.Mock` imported but unused

Remove unused import: `unittest.mock.Mock`

(F401)